### ISSUE TITLE
users not connecting to local server after install

### ISSUE THREAD

**Issue by cybermelons** [tsaibermelon/gg-struggle#4]
## Problem

Game isn't grabbing traffic after install, and errors on start. Then immediately closes.

## What now?

We need to read the error, so we'll run it in its own powershell/cmd window so the window stays up after crashing.

For anyone experiencing this issue, please run gg-struggle in powershell as seen [here](https://twitter.com/cybermelon2/status/1412880858244079616?s=20) and post logs/screenshots here. The logfile can be found at `%temp%/gg-struggle/all.log`

## References:

- [ref1](https://twitter.com/trafopotmeter/status/1412838320535707655?s=20)
- [ref2](https://twitter.com/No94171361/status/1412859251861635073?s=20)

**Comment by brandon-dodds** [comment]
All I did to get it working was run gg-struggle.exe as an administrator. Before that, it wouldn't work and would give the same console output as yours.

**Comment by FogKnight22** [comment]
Unfortunately the traffic doesn't even get into gg-struggle.exe on my end. Firstly, I needed to manually add the lines into the host file because the installer cannot do it automatically (I tried to make it Trusted for my anti-virus but even then it doesn't change anything). I checked if the certificated was imported at least and it was, so there is that. However, nothing was working. The game was booting up as if gg-struggle.exe was not even on (I turned it on before launching the game, just to communicate that).

I then tried PowerShell twice (one while the .exe is running and one without) and still nothing to really discern. Just the first error being that the .exe is already running. I am at a loss for what to do, especially when not even the anti-virus tinkering did anything.

I also tried to Run As Administrator but that didn't change anything either. I really wished the error report could be substantial but it is as if gg-struggle.exe is not even running.

![Capture](https://user-images.githubusercontent.com/86963382/124982108-a2da3f80-e036-11eb-8a0e-db7680385c4e.PNG)
![Capture2](https://user-images.githubusercontent.com/86963382/124982475-095f5d80-e037-11eb-9306-efc7b48f97e9.PNG)
